### ISSUE TITLE
Can you add new even or trigger types?

### ISSUE THREAD

**Issue by UsmanShahzad** [renvrant/conditionize.js#22]
I would like to add keyup as a trigger.

**Comment by rguliev** [comment]
Good idea. I can not promise fast implementation very soon but after the New Year, I will add it with other changes like [this one](https://github.com/renvrant/conditionize.js/issues/20#issuecomment-443425410).

**Comment by rguliev** [comment]
This issue is fixed in the [conditionize2.js](https://github.com/rguliev/conditionize2.js)
Also, take a look at my last comment [here](https://github.com/renvrant/conditionize.js/issues/18#issuecomment-483152192)
